miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SubprojectController.java

```diff
@@ -104,6 +109,12 @@ private static SubprojectDto writeUrls(SubprojectDto subprojectDto, UriComponent
     return subprojectDtos;
   }
 
+  @RequestMapping(value = "/referenceGenomeOptions", method = RequestMethod.GET, produces = { "application/json" })
```

Comment:
To match other resource names, this should be all lower case. To indicate that it's a collection use a plural. "/referencegenomes"